.github/workflows/trivy-security-scan.yml

```diff
@@ -48,38 +73,60 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+      - name: Restore Trivy DB cache
+        uses: actions/cache@v4
+        with:
+          path: ${{ github.workspace }}/.cache/trivy
+          key: cache-trivy-${{ steps.date.outputs.date }}
```

Comment:
When this PR is merged, there won’t be a cache until the scheduled job is first run:
> This event will only trigger a workflow run if the workflow file is on the default branch.

Do we need to manually run the cache job after this PR merges?

Reply:
@teor2345 actually i believe it will run with `trivy-scan-dispatch` event which will create a cache too. Since we run alot of jobs, i will remove the cron schedule.